auth/store.go

```diff
@@ -887,7 +969,7 @@ func (as *authStore) isAuthEnabled() bool {
 	return as.enabled
 }
 
-func NewAuthStore(be backend.Backend, indexWaiter func(uint64) <-chan struct{}) *authStore {
+func NewAuthStore(be backend.Backend, indexWaiter func(uint64) <-chan struct{}, tokenType, jwtSignMethod, jwtPubKeyPath, jwtPrivKeyPath string) *authStore {
```

Comment:
Should be fine if it's public; if external packages want to roll their own provider I think that's OK.